calc.py

```diff
@@ -73,7 +82,7 @@
 
 
 def t_STRING_VALUE(t):
-    r"\"([^\"]*)\""
+    r"\"([^\"]*)\" "
```

Comment:
White space added at the end of regex is intentional? It can change behaviour by expecting whitespace after closing " of a string.